cmd/snap/cmd_routine_portal_info_test.go

```diff
@@ -141,6 +147,67 @@ HasNetworkStatus=true
 	c.Check(s.Stderr(), Equals, "")
 }
 
+func (s *SnapSuite) TestPortalInfoCommonID(c *C) {
+	restore := snap.MockCgroupSnapNameFromPid(func(pid int) (string, error) {
+		c.Check(pid, Equals, 42)
+		return "hello", nil
+	})
+	defer restore()
+	restore = snap.MockApparmorSnapAppFromPid(func(pid int) (string, string, string, error) {
+		c.Check(pid, Equals, 42)
+		return "hello", "common-id", "", nil
+	})
+	defer restore()
+	n := 0
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		switch n {
+		case 0:
+			c.Check(r.Method, Equals, "GET")
+			c.Check(r.URL.Path, Equals, "/v2/snaps/hello")
+			fmt.Fprint(w, mockInfoJSONWithApps)
+		case 1:
+			c.Check(r.Method, Equals, "GET")
+			c.Check(r.URL.Path, Equals, "/v2/connections")
+			c.Check(r.URL.Query(), DeepEquals, url.Values{
+				"snap":      []string{"hello"},
+				"interface": []string{"network-status"},
+			})
+			result := client.Connections{
+				Established: []client.Connection{
+					{
+						Slot: client.SlotRef{
+							Snap: "core",
+							Name: "network-status",
+						},
+						Plug: client.PlugRef{
+							Snap: "hello",
+							Name: "network-status",
+						},
+						Interface: "network-status",
+					},
+				},
+			}
```

Comment:
Can we skip all this part about the network status? It does not seem relevant to the bit of code that you are testing :-)

Reply:
Yeah, done.